EIPS/eip-6049.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6049
+title: Deprecate SELFDESTRUCT
+description: Deprecate SELFDESTRUCT by discouraging its use and warning about a potential future behavior change.
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/deprecate-selfdestruct/11907
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-27
+---
+
+## Abstract
+
+This EIP deprecates the `SELFDESTRUCT` opcode and warns against its use. A breaking change to this functionality is likely to come in the future.
+
+## Motivation
+
+Discussions about how to change `SELFDESTRUCT` are ongoing. But there is a strong desire to do *something*.
+
+## Specification
+
+Documentation of the `SELFDESTRUCT` opcode is updated to warn against its use and to note that a breaking change may be forthcoming.
```

Comment:
Something like this hasn't been done yet. "Deprecation" should be defined.

Reply:
"Deprecation" is not used in the specification.

This spec is referring to updating the language of the Yellow Paper. Right now it doesn't say Yellow Paper since I assume that is the implicit meaning that all core EIPs are an update to the Yellow Paper. But if I can make it more clear, I'll do anything needed here.